cuml/rf_demo.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cuml import RandomForestClassifier as cuRF\n",
```

Comment:
Basic documentation explaining what is happening in the notebook would be nice to have at the beginning of the notebook